### ISSUE TITLE
Have better UX for "user not found" when navigating to a user page that does not exist

### ISSUE THREAD

**Issue by hburn7** [osu-tournament-rating/otr-web#64]
When a user has never played a tournament match, this is what the stats response looks like:

Ensure this response case is handled correctly and the user is informed that they need to play tournaments in order to have their dashboard created.
```json
{
    "generalStats": null,
    "matchStats": {
        "averageMatchCostAggregate": 0,
        "highestRating": 0,
        "highestGlobalRank": 0,
        "highestCountryRank": 0,
        "highestPercentile": 0,
        "ratingGained": 0,
        "gamesWon": 0,
        "gamesLost": 0,
        "gamesPlayed": 0,
        "matchesWon": 0,
        "matchesLost": 0,
        "matchesPlayed": 0,
        "gameWinRate": 0,
        "matchWinRate": 0,
        "averageTeammateRating": null,
        "averageOpponentRating": null,
        "bestWinStreak": 0,
        "matchAverageScoreAggregate": 0,
        "matchAverageMissesAggregate": 0,
        "matchAverageAccuracyAggregate": 0,
        "averageGamesPlayedAggregate": 0,
        "averagePlacingAggregate": 0,
        "mostPlayedTeammateName": null,
        "mostPlayedOpponentName": null,
        "bestTeammateName": null,
        "periodStart": "0001-01-01T00:00:00",
        "periodEnd": "0001-01-01T00:00:00"
    },
    "modStats": {
        "playedNM": null,
        "playedEZ": null,
        "playedHT": null,
        "playedHD": null,
        "playedHR": null,
        "playedDT": null,
        "playedFL": null,
        "playedHDHR": null,
        "playedHDDT": null,
        "playedHDEZ": null
    },
    "tournamentStats": {
        "teamSizeCounts": {
            "count1v1": 0,
            "count2v2": 0,
            "count3v3": 0,
            "count4v4": 0,
            "countOther": 0
        },
        "bestPerformances": [],
        "worstPerformances": []
    },
    "frequentTeammates": [],
    "frequentOpponents": [],
    "ratingStats": []
}
```

**Comment by hburn7** [comment]
Design is final in the figma, implementation pending.

**Comment by hburn7** [comment]
Closed in #190
